crypto/src/main/kotlin/net/corda/v5/crypto/merkle/MerkleTree.kt

```diff
@@ -25,7 +25,7 @@ interface MerkleTree {
     val root: SecureHash
```

Comment:
"@property leaves are the input data elements. Usually the hashes of something larger deterministically serialized."
(line 13)
Suggest:
The input data...